test/presenters/content_item_presenter_test.rb

```diff
@@ -35,6 +35,11 @@ class ContentItemPresenterTest < ActiveSupport::TestCase
     assert_equal expected, ContentItemPresenter.new("schema_name" => "Schema name").related_navigation
   end
 
+  test "#taxonomy_sidebar combines taxonomy links and collections links for the taxonomy sidebar component" do
+    expected = { items: [], collections: [] }
```

Comment:
not sure what this is doing but it looks suspiciously empty so I'm not sure how it proves the test.